test/functional/feature_fee_estimation.py

```diff
@@ -133,12 +133,12 @@ def setup_network(self):
         which we will use to generate our transactions.
         """
         self.add_nodes(3, extra_args=[["-maxorphantx=1000", "-whitelist=127.0.0.1"],
-                                      ["-blockmaxsize=17000", "-maxorphantx=1000"],
-                                      ["-blockmaxsize=8000", "-maxorphantx=1000"]])
+                                      ["-blockmaxweight=68000", "-maxorphantx=1000"],
+                                      ["-blockmaxweight=32000", "-maxorphantx=1000"]])
         # Use node0 to mine blocks for input splitting
         # Node1 mines small blocks but that are bigger than the expected transaction rate.
         # NOTE: the CreateNewBlock code starts counting block size at 1,000 bytes,
-        # (17k is room enough for 110 or so transactions)
+        # (68k weight is room enough for 120 or so transactions)
```

Comment:
I have verified these numbers still correct.

Reply:
Good! Is this test using segwit or legacy transactions? I assume that would have an impact on these numbers?

Reply:
Legacy [P2SH](https://github.com/bitcoin/bitcoin/blob/master/test/functional/feature_fee_estimation.py#L22-L31) - so just a simple `WITNESS_SCALE_FACTOR` for all txs.